unicoremc/models.py

```diff
@@ -173,6 +173,8 @@ class Project(models.Model):
         default=settings.MESOS_DEFAULT_MEMORY_ALLOCATION)
     marathon_instances = models.IntegerField(
         default=settings.MESOS_DEFAULT_INSTANCES)
+    marathon_health_check_path = models.CharField(
+        max_length=255, blank=True, null=True)
```

Comment:
Shouldn't we set a default here `/health/`? It might be easier than having to manually update each project.